atlp-rwanda/elite-bn-fe
# ch(setup-react-project): Setup an empty react project

#### What does this PR do?

Set up an empty react project using Webpack 4, Babel, and the Airbnb linting option.

#### How should this be manually tested?

1. Clone branch, install packages by running or `yarn `.
2. Run `yarn start` to start the server. 
3. Now you can see a page with the **Barefoot Nomad** title opened in your browser.
4. To test linting, run `yarn lint` and fix the lining errors run `yarn lint:fix`

#### What are the relevant pivotal tracker stories?
[#182201515](https://www.pivotaltracker.com/story/show/182201515)
